guide/src/web-getting-started.md

```diff
@@ -0,0 +1,112 @@
+
+## Add the package dependency
+
+### Installing with `npm`
+
+NOTE: Currently you need to build the package locally.
+We intend to publish to npmjs.com very soon.
+
+In your web app, you can add the Ferrostar NPM package as a dependency.
+You will need to install [Rust](https://www.rust-lang.org/) and `wasm-pack` to build the NPM package.
+
+```shell
+cargo install wasm-pack
+```
+
+Head to the local path where you have checked out Ferrostar,
+go to the `web` directory, and build the module:
+
+```shell
+npm install && npm run build
+```
+
+Then, in your project, install the Ferrostar package using the local path:
+
+```shell
+npm install /path/to/ferrostar/web
+```
+
+### Using unpkg
+
+TODO after publishing to npm.
+
+## Add Ferrostar web components to your web app
+
+The Ferrostar web SDK uses the [Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components)
+to ensure maximum compatibility across frontend frameworks.
+You can import the components just like other things you’re used to in JavaScript.
+
+```javascript
+import { FerrostarCore, BrowserLocationProvider } from "ferrostar-components";
+```
+
+## Configure the `<ferrostar-core>` component
+
+Now you can use Ferrostar in your HTML like this:
+
+```html
+<ferrostar-core
+  id="core"
+  valhallaEndpointUrl="https://api.stadiamaps.com/route/v1"
+  styleUrl="https://tiles.stadiamaps.com/styles/outdoors.json"
+  profile="bicycle"
+  useIntegratedSearchBox="true"
+></ferrostar-core>
+```
+
+Here we have used Stadia Maps URLs, which should work without authentication for local development.
+(Refer to the [authentication docs](https://docs.stadiamaps.com/authentication/)
+for network deployment details; you can start with a free account.)
+
+See the [vendors appendix](./vendors.md) for a list of other compatible vendors.
+
+`<ferrostar-core>`  additionally requires setting some CSS manually, or it will be invisible!
+
+```css
+ferrostar-core {
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+```
+
+That’s all you need to get started!
+
+### Configuration explained
+
+`<ferrostar-core>` provides a few properties to configure.
+Here are the most important ones:
+
+- `valhallaEndpointUrl`: The Valhalla routing endpoint to use. You can use any reasonably up-to-date Valhalla server, including your own. See [vendors](./vendor.md#routing) for a list of known compatible vendors.
+- `httpClient`: You can set your own fetch-compatible HTTP client to make requests to the routing API (ex: Valhalla).
+- `costingOptions`: You can set the costing options for the route provider (ex: Valhalla JSON options).
+- `useIntegratedSearchBox`: Ferrostar web includes a search box powered by Stadia Maps, but you can disable this and replace with your own.
+- `useVoiceGuidance`: Enable or disable voice guidance.
+
+NOTE: `useIntegratedSearchBox` and `useVoiceGuidance` are disabled by default. Set them to any value to enable them.
```

Comment:
This is a [limitation](https://lit.dev/docs/components/properties/#boolean-attributes) of Lit.

Reply:
Makes sense; thanks for linking that + clarifying in the docs!